src/PictureCard/index.js

```diff
@@ -0,0 +1,230 @@
+  );
+`;
+
+StyledOverlay.defaultProps = {
+  theme: defaultTheme,
+};
+
+const StyledOverlayHover = styled.div`
+  ${overlayCss};
+  opacity: 0;
+  background: rgba(0, 0, 0, 0.8);
+`;
+
+StyledOverlayHover.defaultProps = {
+  theme: defaultTheme,
+};
+
+const StyledPictureCardContent = styled.div`
+  position: relative;
+  z-index: 2;
+  display: flex;
+  flex-direction: column;
+  padding: ${({ theme }) => theme.orbit.spaceMedium};
+  transition: bottom ${({ theme }) => theme.orbit.durationFast} ease-in-out;
+  bottom: ${({ contentHeight, theme }) =>
+    contentHeight && `-${contentHeight + parseInt(theme.orbit.spaceMedium, 10)}px`};
+`;
+
+StyledPictureCardContent.defaultProps = {
+  theme: defaultTheme,
+};
+
+const StyledPictureCard = styled(({ height, href, theme, external, ...props }) => {
+  const Component = href ? "a" : "div";
+  return <Component {...props}>{props.children}</Component>;
+})`
+  height: ${({ height }) => (height ? `${height}px` : "100%")};
+  width: ${({ width }) => (width ? `${width}px` : `100%`)};
+  display: flex;
+  align-items: flex-end;
+  position: relative;
+  box-sizing: border-box;
+  border-radius: ${({ theme }) => theme.orbit.borderRadiusNormal};
+  overflow: hidden;
+  cursor: pointer;
+  box-shadow: ${({ theme }) => theme.orbit.boxShadowAction};
+
+  &:hover {
+    box-shadow: ${({ theme, shadows }) => shadows && theme.orbit.boxShadowActionActive};
+    ${StyledOverlayHover} {
+      opacity: 1;
+    }
+
+    ${StyledPictureCardContent} {
+      bottom: 0;
+    }
+
+    ${Shown} {
+      opacity: 1;
+      top: 0;
+    }
+  }
+
+  &:focus {
+    outline: none;
+    box-shadow: ${({ theme }) => theme.orbit.boxShadowActionActive};
+  }
+`;
+
+StyledPictureCard.defaultProps = {
+  theme: defaultTheme,
+};
+
+const PictureCard = ({
+  onClick,
+  image,
+  title,
+  tabIndex,
+  subTitle,
+  href,
+  children,
+  external,
+  actions,
+  label,
+  height = 300,
+  width,
+  dataTest,
+}: Props) => {
+  const [contentHeight, setContentHeight] = React.useState(0);
+  const ref: { current: any | HTMLDivElement } = React.useRef(null);
+
+  React.useEffect(() => {
+    if (ref.current) {
+      setContentHeight(ref.current.clientHeight);
+    }
+  }, [setContentHeight]);
+
+  const cardID = React.useMemo(() => randomID("destinationCardID"), []);
+
+  const handleKeyDown = (ev: SyntheticKeyboardEvent<HTMLElement>) => {
+    if (onClick) {
+      if (ev.keyCode === KEY_CODE_MAP.ENTER) {
+        onClick();
+      } else if (ev.keyCode === KEY_CODE_MAP.SPACE) {
+        ev.preventDefault();
+        onClick();
+      }
+    }
+  };
+
+  const { name, original, placeholder } = image;
+
+  return (
+    <StyledPictureCard
+      data-test={dataTest}
+      onClick={onClick}
+      onKeyDown={handleKeyDown}
+      height={height >= SMALLEST_HEIGHT ? height : SMALLEST_HEIGHT}
+      width={width}
+      href={href}
+      external={external}
+      shadows={onClick || href}
+      tabIndex={tabIndex}
```

Comment:
I believe that `tabIndex` should be always `"0"` when `href` is NOT specified. Otherwise it won't be focusable because it's a div.